capstone/src/main/webapp/text_analyzer.html

```diff
@@ -33,7 +33,15 @@ <h3 class="masthead-brand">Perspective Hub</h3>
           </nav>
         </div>
       </header>
-      <h2>Input text and select the Perspective API attribute(s) you would like to be shown</h2>
+      <main role="main">
+        <section class="jumbotron text-center">
+          <div class="container">
+            <h1>Text Analysis</h1>
+            <p class="lead text-muted">Input text and select the Perspective API attribute(s) you would like to be shown</p>
+            <small id="disclaimer" class="form-text text-muted">Copy and paste supported</small>
```

Comment:
nit: Are we sure this detail should be included? I would assume the user would know this is inferred everywhere they can write something